test/integration/test_resolvers.py

```diff
@@ -0,0 +1,24 @@
+"""Integration tests for the conda dependency resolution."""
+
+import os
+from base import integration_util
+from base.api import ApiTestCase
+
+GNUPLOT = {u'version': u'4.6', u'type': u'package', u'name': u'gnuplot'}
+
+class CondaResolutionIntegrationTestCase(integration_util.IntegrationTestCase, ApiTestCase):
```

Comment:
Update 2: Importing svgwrite before twill in test/base/twilltestcase.py is the exact point where this breaks down. Something in twill prevents svgwrite from working.